### PR TITLE
Feat/account page

### PR DESCRIPTION
**Pull request by FahdW**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [ ] I've added this PR's link to its Asana task(s)
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

### CODE REVIEW

**Comment by remon-nashid** [comment]
@FahdW is this is ready for review?

**Comment by FahdW** [comment]
Yeah, go ahead on the review. Design wise it could be a lot better. But for simplicity sake this is what is there atm

File: packages/polymath-issuer/src/pages/account/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  render() {
+    const { tickers } = this.props;
+    return (
+      <PageCentered>
+        <Grid>
+          <Grid.Row>
+            <p>
+              <Link to="/ticker">
+                <Button id="create-token-btn" icon="arrow--right">
+                  Continue to create your security token
```
Comment (remon-nashid) [inline packages/polymath-issuer/src/pages/account/index.js]: @FahdW per our most recent changes in `develop`, the word "Create" should be replaced with "Configure"

File: packages/polymath-issuer/src/pages/home/index.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ class HomePage extends Component {
           <br />
           <br />
           <p>
-            <Link to="/ticker">
+            <Link to="/account">
               <Button id="create-token-btn" icon="arrow--right">
                 Create your security token
```
Comment (remon-nashid) [inline packages/polymath-issuer/src/pages/home/index.js]: r/Create/Configure

**Comment by FahdW** [comment]
> @FahdW there's a glitsh in navigation. To reproduce:
> 
> * Click on "create your ST"
> * Then click on "Continue"
> * Click on browser's back button to go back to `/account` page.
> * The url changes to `/account` but token creation page remains

Appears that this glitch is throughout our app. It maybe the way that our router is done.

**Pull request merged**